apps/rotm/emails/caseworker.html

```diff
@@ -0,0 +1,25 @@
+<h1>A Report on terrorism material has been made</h1>
+
+{{#data}}
+  <table width="100%" cellspacing="0" cellpadding="0" rules="none">
+    {{#table}}
+      {{#linebreak}}
+      <tr>
+        <td colspan="2" style="border-bottom:1px solid #333;"></td>
+      </tr>
+      {{/linebreak}}
+      {{#value}}
+      <tr>
+        <td width="40%" style="padding: 15px 0;border-bottom:1px solid #ccc;">{{label}}</td>
+        <td width="60%" style="padding: 15px 0;border-bottom:1px solid #ccc;">{{value}}</td>
+      </tr>
+      {{/value}}
+    {{/table}}
+    <tr>
+      {{#image-url}}
+        <td width="40%" style="padding: 15px 0;border-bottom:1px solid #ccc;">Image</td>
+        <td width="60%" style="padding: 15px 0;border-bottom:1px solid #ccc;"><a href="{{image-url}}">View the image</a></td>
```

Comment:
As described above, image link is to be displayed differently to the other data because of the href